sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_options.hpp

```diff
@@ -43,55 +43,23 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
     Azure::Nullable<std::string> NextPageToken;
   };
 
-  class ServiceVersion final {
```

Comment:
This will be considered a breaking change, so we should really explore why it's worth doing.

Reply:
Jeff wants this, it makes sense to have it this way. Having this struct and the versions defined implies we are testing and guaranteeing that thhe SDK works with other versions than 7.3 which is false, we don't even run the unit tests with previous versions. WE DO NOT GUARANTEE THAT OTHER VERSIONS THAN THE CURRENT ONE WORK.